listenbrainz/webserver/views/stats_api.py

```diff
@@ -587,21 +587,91 @@ def get_sitewide_artist():
+                        "artist_name": "Ellie Goulding",
+                        "listen_count": 25,
+                        "release_mbid": "",
+                        "release_name": "Delirium (Deluxe)"
+                    },
+                    {
+                        "artist_mbids": [],
+                        "artist_name": "The Fray",
+                        "listen_count": 25,
+                        "release_mbid": "",
+                        "release_name": "How to Save a Life"
+                    },
+                ],
+                "offset": 0,
+                "count": 2,
+                "range": "year",
+                "last_updated": 1588494361,
+                "from_ts": 1009823400,
+                "to_ts": 1590029157
+            }
+        }
+
+    .. note::
+        - This endpoint is currently in beta
+        - ``artist_mbids``, ``artist_msid``, ``release_mbid`` and ``release_msid`` are optional fields and
+          may not be present in all the responses
+
+    :param count: Optional, number of artists to return for each time range,
+        Default: :data:`~webserver.views.api.DEFAULT_ITEMS_PER_GET`
+        Max: :data:`~webserver.views.api.MAX_ITEMS_PER_GET`
+    :type count: ``int``
+    :param offset: Optional, number of artists to skip from the beginning, for pagination.
+        Ex. An offset of 5 means the top 5 artists will be skipped, defaults to 0
+    :type offset: ``int``
+    :param range: Optional, time interval for which statistics should be returned, possible values are
+        :data:`~data.model.common_stat.ALLOWED_STATISTICS_RANGE`, defaults to ``all_time``
+    :type range: ``str``
+    :statuscode 200: Successful query, you have data!
+    :statuscode 204: Statistics haven't been calculated, empty response will be returned
+    :statuscode 400: Bad request, check ``response['error']`` for more details
+    :resheader Content-Type: *application/json*
+    """
+    return _get_sitewide_stats("releases")
+
+
+def _get_sitewide_stats(entity: str):
+    stats_range = request.args.get("range", default="all_time")
     if not _is_valid_range(stats_range):
-        raise APIBadRequest("Invalid range: {}".format(stats_range))
+        raise APIBadRequest(f"Invalid range: {stats_range}")
 
-    offset = get_non_negative_param('offset', default=0)
-    count = get_non_negative_param('count', default=DEFAULT_ITEMS_PER_GET)
+    offset = get_non_negative_param("offset", default=0)
+    count = get_non_negative_param("count", default=DEFAULT_ITEMS_PER_GET)
 
-    stats = db_stats.get_sitewide_stats(stats_range, 'artists')
+    stats = db_stats.get_sitewide_stats(stats_range, entity)
     if stats is None:
-        raise APINoContent('')
+        raise APINoContent("")
 
     entity_list, total_entity_count = _process_user_entity(stats, offset, count)
     return jsonify({
         "payload": {
-            "artists": entity_list,
+            entity: entity_list,
```

Comment:
Missing quotes?

Reply:
Ah no, its intended. This method (see line 659) take entity variable as input which contains the value artists, releases or recordings etc. So the payload will have keys according to the stat endpoint called. For artists, it will 
`{ "payload": { "artists" : [] }}`. For releases like  `{ "payload": { "releases" : [] }}`. So using the value of entity variable here as the key.

(FWIW, I don't like changing the payload structure like this. I'd prefer another key which contains the name of the stat returned but this is what we do for user stats and having two slightly different formats for stats is probably more confusing so this is what we are stuck with for the time being.)